sql-plugin/src/main/scala/com/nvidia/spark/rapids/RegexParser.scala

```diff
@@ -565,7 +565,8 @@ class CudfRegexTranspiler(mode: RegexMode) {
       case RegexChar(ch) => ch match {
         case '.' =>
           // workaround for https://github.com/rapidsai/cudf/issues/9619
-          RegexCharacterClass(negated = true, ListBuffer(RegexChar('\r'), RegexChar('\n')))
+          val terminatorChars = ListBuffer('\r', '\n', '\u0085', '\u2028', '\u2029')
+          RegexCharacterClass(negated = true, terminatorChars.map(RegexChar))
```

Comment:
`terminatorChars` is unnecessary here, there is already a local private val called `lineTerminatorChars` which has the same values in a Seq.

```suggestion
          RegexCharacterClass(negated = true, lineTerminatorChars.map(RegexChar))
```

Reply:
I had that at first but it would change the ordering of the line terminators. It would convert `. ` to `[^\n\r ... ]` instead of `[^\r\n ... ]`. It shouldn't make a difference but because of the way some of the tests check, tests would fail.

Reply:
I think it would only change a few of the transpilation tests, but I don't think it would be too hard to change. The order within the character class also doesn't matter for the purposes of the regular expression, so you could do this and minimize how many tests to update.